### PR TITLE
Support travis and last jdk build

### PR DESCRIPTION
**Pull request by jbescos**
image

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+dist: trusty
```
Comment (lukasj) [inline .travis.yml:1]: use latest available, should be `focal`

**Comment by pzygielo** [comment]
Please mind that all free ee4j's credits are already consumed at travis.com.

**Comment by lukasj** [comment]
right. Note that some ee4j projects are still on travis.org, even though that is not the case here.

**Review by lukasj — COMMENTED** [review]
add license headers to new files

**Review by lukasj — APPROVED** [review]
LGTM

**Comment by pzygielo** [comment]
I don't understand.  I can't see the hidden goal of this PR, as definitely it's not to run builds on travis.

**Comment by lukasj** [comment]
> I don't understand. I can't see the hidden goal of this PR, as definitely it's not to run builds on travis.

https://travis-ci.org/github/lukasj/mail/builds/773471211 is not on travis?

**Comment by pzygielo** [comment]
> https://travis-ci.org/github/lukasj/mail/builds/773471211 is not on travis?

For next 11 days and in fork.

**Comment by lukasj** [comment]
so can you come up and set up something else?

**Comment by pzygielo** [comment]
> so can you come up and set up something else?

It's still not clear what's the goal here. But perhaps #563?

**Pull request merged**